filter/store.go

```diff
@@ -41,11 +43,28 @@ type HTTPClient interface {
 }
 
 // PutCSVData allows the filtered file data to be sent back to the filter store when complete.
-func (store *Store) PutCSVData(filterJobID string, url string, size int64) error {
+func (store *Store) PutCSVData(filterJobID string, fileURL string, size int64) error {
 
-	// todo: make call to the filter API.
+	url := store.filterAPIURL + "/filters/" + filterJobID
+
+	putBody := &observation.Filter{
```

Comment:
Not sure the observation package is correctly named? A filter contains many observations not the other way around? O.o Haven't looked into what all the observation package is doing to know a better name though